internal/services/configstore/db/migrate.go

```diff
@@ -115,3 +116,33 @@ func (d *DB) migrateV2(tx *sql.Tx) error {
 
 	return nil
 }
+
+func (d *DB) migrateV3(tx *sql.Tx) error {
+	var ddlPostgres = []string{
+		"ALTER TABLE project ADD COLUMN members_can_perform_run_actions boolean NOT NULL DEFAULT false",
+		"ALTER TABLE project ALTER COLUMN members_can_perform_run_actions DROP DEFAULT",
```

Comment:
@sgotti in the first statement if I remove `DEFAULT false` the error is `pq: column "can_members_perform_run_actions" of relation "project" contains null values`, because we have already sono project rows in db.
In the second statement I remove DEFAULT to have the same shema of V3.

Reply:
Ok, It just doesn't looks very clear. I'll prefer to do this in the standard 3 ways steps: Add column without non null constraint, update column with default values, add null constraint to the column.